transfers/transfer.py

```diff
@@ -463,13 +507,16 @@ def main(am_user, am_api_key, ss_user, ss_api_key, ts_uuid, ts_path, depth, am_u
         return 0
     # If waiting on input, send email, exit
     elif status == 'USER_INPUT':
-        LOGGER.info('Waiting on user input, running scripts in user-input directory.')
+        LOGGER.info(
+            'Waiting on user input, running scripts in user-input directory.')
         # TODO What inputs do we want?
         microservice = status_info.get('microservice', '')
         run_scripts(
             'user-input',
             microservice,  # Current microservice name
-            str(microservice != current_unit.microservice),  # String True or False if this is the first time at this wait point
+            str(microservice != current_unit.microservice),
+            # String True or False if this is the first time at this wait
+            # point
```

Comment:
Nitpick: I think the descriptive comment should go above the thing it describes.